server/models/blocked.js

```diff
@@ -46,5 +37,42 @@ module.exports = (sequelize, { STRING }) => {
     });
   };
 
+  Blocked.refreshTable = function () {
+    let blockedList = [];
+    return fetch('https://api.lbry.io/file/list_blocked')
+      .then(response => {
+        return response.json();
+      })
+      .then(jsonResponse => {
+        if (!jsonResponse.data) {
+          throw new Error('no data in list_blocked response');
+        }
+        if (!jsonResponse.data.outpoints) {
+          throw new Error('no outpoints in list_blocked response');
+        }
+        return jsonResponse.data.outpoints;
+      })
+      .then(outpoints => {
+        logger.debug('total outpoints:', outpoints.length);
+        // prep the records
+        for (let i = 0; i < outpoints.length; i++) {
+          blockedList.push({
+            outpoint: outpoints[i],
+          });
+        }
+        // clear the table
+        return this.destroy({
+          truncate: true,
+        });
+      })
+      .then(() => {
+        // fill the table
+        return this.bulkCreate(blockedList);
+      })
+      .catch(error => {
+        throw error;
```

Comment:
Instead of throwing, catching, and throwing an error again, you can just skip this `catch` block and it will go to the next highest `catch`